test/osmt-apitest.env.example

```diff
@@ -0,0 +1,4 @@
+OKTA_URL=https://dev-XXXXXXXX.okta.com
+LOCAL_URL=http://localhost:8080
```

Comment:
Seems should BASE_URL

Reply:
Yes, that is correct. In the files it is BASE_URL just forgot to change it in the env.example file. Good catch

Reply:
Marking this conversation as 'unresolved' until the change is in the PR